presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSessionProperties.java

```diff
@@ -129,4 +135,9 @@ public static DataSize getStorageBasedBroadcastJoinWriteBufferSize(Session sessi
     {
         return session.getSystemProperty(STORAGE_BASED_BROADCAST_JOIN_WRITE_BUFFER_SIZE, DataSize.class);
     }
+
+    public static DataSize getPosBroadcastJoinMaxMemory(Session session)
```

Comment:
We don't use `pos` acronym anywhere in the code. We prefer to use `PrestoSpark` /  `Spark` instead (e.g.:  `spark_partition_count_auto_tune_enabled `, `min_spark_input_partition_count_for_auto_tune `, ... ) 

Also since this is effectively a "hard override" I recommend calling it out explicitly in the name, say `getSparkBroadcastJoinMaxMemoryOverride` (`spark_broadcast_join_max_memory_override`, `SPARK_BROADCAST_JOIN_MAX_MEMORY_OVERRIDE`).